### PR TITLE
build: add codespace config

### PR DESCRIPTION
**Pull request by aryarm**
This PR adds a convenient method to work with haptools in the cloud: [GitHub codepsaces](https://docs.github.com/en/codespaces)! This can be helpful when you want to quickly play around with our code but don't want to go through all of the setup steps. Just click on the "Open in Github Codespaces" badge on the README and voila - you'll be presented with a VS Code environment automagically set up with haptools and our development dependencies. (Note: you might need to run `conda activate haptools` to access it.) This environment will run with (usually free-for-academic-use) credits on Github's codespace VMs.

### CODE REVIEW

**Comment by aryarm** [comment]
@mlamkin7, wanna try this out before I merge?
I've become a big fan of codespaces recently, so I'm trying to set that up for us everywhere :)

**Comment by mlamkin7** [comment]
> @mlamkin7, wanna try this out before I merge? I've become a big fan of codespaces recently, so I'm trying to set them up for us everywhere :)

Will do.

File: README.md
Comment (mlamkin7) [inline README.md:1]: I can't see the changes in the readme on the branch, so I can't mess around with codespaces right now.

Comment (aryarm) [reply README.md:1]: ah! try this link to [the new badge on the `build/codespace` branch](https://github.com/CAST-genomics/haptools/tree/build/codespace#haptools)) and let me know if it works for you

or alternatively, just go directly to the codespaces menu:
https://github.com/codespaces/new?skip_quickstart=true&machine=basicLinux32gb&repo=386377111&ref=build%2Fcodespace&geo=UsWest&devcontainer_path=.devcontainer.json

[](https://codespaces.new/CAST-genomics/haptools?ref=build%2Fcodespace)

Comment (mlamkin7) [reply README.md:1]: Am I going to have to pay for launching the machine?

Comment (aryarm) [reply README.md:1]: It should be free for academic use. @mlamkin7, have you signed up for [GitHub education](https://education.github.com/students) already?

**Review by mlamkin7 — APPROVED** [review]

**Pull request merged**